src/models/configurationSettings.ts

```diff
@@ -1,4 +1,4 @@
-"use strict";
+/// <reference path="../../typings/vscode-typings.d.ts" />

```

Comment:
@Huachao, you are right. Explicit typings are only necessary for npm packages that don't contain them out of the box, as it seems (http://www.typescriptlang.org/docs/handbook/typings-for-npm-packages.html). I'm going to save a few lines of code from now on ;)